src/kiwixapp.h

```diff
@@ -83,6 +83,7 @@ class KiwixApp : public QtSingleApplication
     QString getText(const QString &key) { return m_translation.getText(key); };
 
     bool isCurrentArticleBookmarked();
+    void setFullScreenDisable(bool disable) {mpa_actions[ToggleFullscreenAction]->setDisabled(disable);}
```

Comment:
I don't think we need to introduce this new method here. It's better (more readable, even longer) already used approach:
```c
KiwixApp::instance()->getAction(KiwixApp::ToggleFullscreenAction)
    ->setDisabled(true);
```